src/main/java/com/rocketden/tester/service/generators/JavaDriverGeneratorService.java

```
@@ -79,4 +83,68 @@ public void writeToStringCode() {
         // TODO Auto-generated method stub
 
     }
+
+    @Override
+    public String typeInstantiationToString(ProblemIOType ioType) {
+        if (ioType == null) {
+            return null;
```

Maybe throw an exception here and in default?

---

Sounds good! 👍